### DISCUSSION TITLE
calling rpc function with date argument from flutter give error

### DISCUSSION THREAD

**Discussion by SaddamMohsen** [supabase/supabase#18967] [Questions] [answered]
i have this postgres function in supabase 
`get_daily_report(f_id int,amb_id int,repDate date)`
that return a record 
the problem not here it work perfectly in sql editor 
but when i calling it from a flutter with supabaseClient like this
`supabaseClient.rpc('get_daily_report',params:{'f_id':1,'amb_id':1,'repDate':Datetime.Now()})`

it give me error 
and the error caused by rep_date parameter 
i tried other functions that doesn't contain a date as argument and they are working 
so how i can solve the problem of passing datetime argument into postgres function 
i tried a lot of solution like make date argument string then convert it inside function into date even this not working 

thank in advance  for your help

**Comment by GaryAustin1** [comment]
I don't know flutter.  In JS we have to do:  `(new Date()).toISOString()`

**Reply by SaddamMohsen** [reply, marked as answer]
Thank you for response
I solved the problem, and it turns out that the problem wasn't with how the date was written. Instead, I found that function and variable names are converted to lowercase when they're converted to JSON for sending over the internet.

**Comment by SaddamMohsen** [comment]
I solved the problem, and it turns out that the problem wasn't with how the date was written. Instead, I found that function and variable names are converted to lowercase when they're converted to JSON for sending over the internet.
